src/Platform/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+set(public_files
+    "${CMAKE_CURRENT_SOURCE_DIR}/include/OpenLoco/Platform/Crash.h"
+    "${CMAKE_CURRENT_SOURCE_DIR}/include/OpenLoco/Platform/Platform.h"
+)
+
+set(private_files
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/Crash.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/Platform.Posix.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/Platform.Windows.cpp"
+)
+
+if (APPLE)
+    set(private_files 
+        "${CMAKE_CURRENT_SOURCE_DIR}src/Platform/Platform.Macos.mm")
```

Comment:
extra `Platform` in the address and you are missing a `/`

Reply:
Oh and also this should append to the private_files not set the private files.

Reply:
I took this as is from the OpenLoco CMakeLists didn't give it much thought, I'll look into it.